presentation/backend/src/presentation-backend/Utils.ts

```diff
@@ -13,17 +13,20 @@ import {
   combineDiagnosticsSeverities, compareDiagnosticsSeverities, Diagnostics, DiagnosticsLogEntry, DiagnosticsOptions, InstanceKey,
 } from "@itwin/presentation-common";
 
-const ecPresentation = require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json"); // eslint-disable-line @typescript-eslint/no-var-requires
-const rulesEngine = require("@itwin/presentation-common/lib/cjs/assets/locales/en/RulesEngine.json"); // eslint-disable-line @typescript-eslint/no-var-requires
+const presentation = require("@itwin/presentation-common/lib/cjs/assets/locales/en/Presentation.json"); // eslint-disable-line @typescript-eslint/no-var-requires
 
 /** @internal */
 export function getLocalizedStringEN(key: string) {
+  let result = presentation;
   const [namespace, identifier] = key.split(":", 2);
-  if (namespace === "ECPresentation") {
-    return ecPresentation[identifier];
-  }
-  if (namespace === "RulesEngine") {
-    return rulesEngine[identifier];
+  if (namespace === "Presentation") {
```

Comment:
I think it would be nice to invert this `if` statement to return if namespace is not `Presentation`. In that case `for` loop can be indented less.